### PR TITLE
Ignore config.yml to have better caching

### LINKED ISSUE #33625: config.yml has absolute path in test of integration-tests/gradle
### Describe the bug

link: https://ge.quarkus.io/c/nk5vgua7gpx2u/k4ldeu42numus/goal-inputs?cacheability=cacheable&expanded=WyI3Y3dyYmU0bGY2dXprLWFubm90YXRpb25wcm9jZXNzaW5ncmVzb3VyY2VzIiwiN2N3cmJlNGxmNnV6ay1hbm5vdGF0aW9uUHJvY2Vzc2luZ1Jlc291cmNlcy0wLTAiLCI3Y3dyYmU0bGY2dXprLWFubm90YXRpb25wcm9jZXNzb3JwYXRoIiwiN2N3cmJlNGxmNnV6ay1hbm5vdGF0aW9uUHJvY2Vzc29yUGF0aC0xLTAiXQ#7cwrbe4lf6uzk

.quarkus/config.yaml ends up with:

```
---
debug: true
registries:
- test.registry.quarkus.io:
    maven:
        repository:
            url: file:///Users/manderse/code/quarkusio/gradle-enterprise-maven-build-validation/.data/02-validate-local-build-caching-different-locations/20230526T083333-647052bd/first-build_quarkus/integration-tests/gradle/target/classes/test-registry-repo
```

### Expected behavior

not sure it is avoidable but if not we should mark it as ignorable.

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by maxandersen**
fixes #33625

### CODE REVIEW

File: .mvn/gradle-enterprise-custom-user-data.groovy [hunk 1]
```diff
@@ -17,7 +17,7 @@ if (System.env.GITHUB_ACTIONS) {
     
    
     def prnumber = System.env.PULL_REQUEST_NUMBER
-    if (prnumber != null) {
+    if (prnumber != null || prnumber.trim() != '') {
```
Comment (gsmet) [inline .mvn/gradle-enterprise-custom-user-data.groovy:20]: It should be `&&`, right? Not sure about `isBlank()` but I suppose it's also available there?
```suggestion
    if (prnumber != null && !prnumber.isBlank()) {
```

File: docs/src/main/asciidoc/_includes/devtools/build-native-container-parameters.adoc [hunk 2]
```diff
@@ -18,4 +18,11 @@ ifdef::devtools-wrapped[+]
...
 ----
-endif::[]
\ No newline at end of file
+endif::[]
+
+ifdef::dev-support-disclaimer[+]
+{dev-support-disclaimer}
+endif::[]
+
+[NOTE]:  Quarkus developemnt tools, including The Quarkus Maven Plugin, Quarkus CLI & Quarkus Gradle Plugin are not supported for deploying your workloads into production environemnts
```
Comment (gsmet) [inline docs/src/main/asciidoc/_includes/devtools/build-native-container-parameters.adoc:27]: I don't think this is related to this PR?

File: docs/src/main/asciidoc/javascript/asciidoc-tabs.js [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 $(document).ready(function() {
     function addBlockSwitches() {
-        $('.listingblock.primary, .sidebarblock.primary').each(function() {
+        $('.xlistingblock.primary, .xsidebarblock.primary, .ulist.primary').each(function() {
```
Comment (gsmet) [inline docs/src/main/asciidoc/javascript/asciidoc-tabs.js:7]: Same here.

**Review by gsmet — CHANGES_REQUESTED** [review]

File: integration-tests/gradle/pom.xml [hunk 4]
```diff
@@ -19,6 +19,28 @@
         <mapstruct.version>1.4.2.Final</mapstruct.version>
     </properties>
 
+    <pluginManagement>
```
Comment (gsmet) [inline integration-tests/gradle/pom.xml:22]: This should be inside a `<build>` section.

File: integration-tests/gradle/pom.xml [hunk 5]
```diff
@@ -19,6 +19,28 @@
...
+            <!-- fixes https://github.com/quarkusio/quarkus/issues/33625 -->
+            <plugin>
+            <groupId>com.gradle</groupId>
+            <artifactId>gradle-enterprise-maven-extension</artifactId>
+            <configuration>
+                <gradleEnterprise>
+                <normalization>
+                    <runtimeClassPath>
+                    <ignoredFiles>
+                        <ignoredFile>META-INF/build.properties</ignoredFile>
```
Comment (gsmet) [inline integration-tests/gradle/pom.xml:34]: The PR title says you are ignoring `config.yml` but it doesn't look like this is what you're doing? Or is `config.yml` a part of `build.properties`?

**Review by gsmet — CHANGES_REQUESTED** [review]

File: .mvn/gradle-enterprise-custom-user-data.groovy (hunk 1 above)
Comment (maxandersen) [reply .mvn/gradle-enterprise-custom-user-data.groovy:20]: Doh. Pushed the wrong commit. Will redo.

File: docs/src/main/asciidoc/_includes/devtools/build-native-container-parameters.adoc (hunk 2 above)
Comment (maxandersen) [reply docs/src/main/asciidoc/_includes/devtools/build-native-container-parameters.adoc]: correct. pushed update with right commits.

**Review by gsmet — APPROVED** [review]
I force pushed to fix the formatting. LGTM!

**Pull request merged**